### PR TITLE
Using separate selendroid installer package

### PR DESCRIPTION
**Pull request by sebv**
Seem to work for me, ping @Jonahss @jlipps

### CODE REVIEW

**Comment by Jonahss** [comment]
Looks good to me, except for the `TODO` with MD5, but that could be done in a separate commit.

File: lib/driver.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import path from 'path';
 import ADB from 'appium-adb';
 import { BaseDriver } from 'appium-base-driver';
 import { SelendroidServer } from './selendroid';
-import { serverExists } from './setup';
+import { serverExists } from './appium-selendroid-installer';
```
Comment (Jonahss) [inline lib/driver.js]: not `./appium-selendroid-installer` but `appium-selendroid-installer`

**Comment by Jonahss** [comment]
it downloads selendroid, but tests don't pass because of my comment above. Just a typo in a require statement I think.

**Comment by jlipps** [comment]
@sebv it looks like you based this off of the setup.js before I made my appium-support changes. Might want to base it off the new setup.js (where I already got rid of the md5 thing)

**Pull request merged**